From 3ed973febb704a73ab4b38b9c3da61e0d8604f6c Mon Sep 17 00:00:00 2001
From: Shaohui Xie <Shaohui.Xie@freescale.com>
Date: Fri, 29 Jan 2016 11:39:40 +0800
Subject: [PATCH 125/666] powerpc/85xx: Add bman_fbpr & qman_fqd & qman_pfdr
 for some Socs

Also remove duplicated bportals.

Signed-off-by: Shaohui Xie <Shaohui.Xie@freescale.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/boot/dts/fsl/p2041rdb.dts      |  4 ----
 arch/powerpc/boot/dts/fsl/p2041si-post.dtsi | 10 ++++++++++
 arch/powerpc/boot/dts/fsl/p3041ds.dts       |  4 ----
 arch/powerpc/boot/dts/fsl/p3041si-post.dtsi | 10 ++++++++++
 arch/powerpc/boot/dts/fsl/p4080ds.dts       |  4 ----
 arch/powerpc/boot/dts/fsl/p4080si-post.dtsi | 10 ++++++++++
 arch/powerpc/boot/dts/fsl/p5020ds.dts       |  4 ----
 arch/powerpc/boot/dts/fsl/p5020si-post.dtsi | 10 ++++++++++
 arch/powerpc/boot/dts/fsl/p5040ds.dts       |  4 ----
 arch/powerpc/boot/dts/fsl/p5040si-post.dtsi | 10 ++++++++++
 10 files changed, 50 insertions(+), 20 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/p2041rdb.dts b/arch/powerpc/boot/dts/fsl/p2041rdb.dts
index 0015f75..e5ffe64 100644
--- a/arch/powerpc/boot/dts/fsl/p2041rdb.dts
+++ b/arch/powerpc/boot/dts/fsl/p2041rdb.dts
@@ -81,10 +81,6 @@
 		ranges = <0x00000000 0xf 0x00000000 0x01008000>;
 	};
 
-	bportals: bman-portals@ff4000000 {
-		ranges = <0x0 0xf 0xf4000000 0x200000>;
-	};
-
 	qportals: qman-portals@ff4200000 {
 		ranges = <0x0 0xf 0xf4200000 0x200000>;
 	};
diff --git a/arch/powerpc/boot/dts/fsl/p2041si-post.dtsi b/arch/powerpc/boot/dts/fsl/p2041si-post.dtsi
index 4bfe1b8..2569dd2 100644
--- a/arch/powerpc/boot/dts/fsl/p2041si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/p2041si-post.dtsi
@@ -47,6 +47,16 @@
 	alloc-ranges = <0 0 0x10 0>;
 };
 
+&qman_fqd {
+	compatible = "fsl,qman-fqd";
+	alloc-ranges = <0 0 0x10 0>;
+};
+
+&qman_pfdr {
+	compatible = "fsl,qman-pfdr";
+	alloc-ranges = <0 0 0x10 0>;
+};
+
 &lbc {
 	compatible = "fsl,p2041-elbc", "fsl,elbc", "simple-bus";
 	interrupts = <25 2 0 0>;
diff --git a/arch/powerpc/boot/dts/fsl/p3041ds.dts b/arch/powerpc/boot/dts/fsl/p3041ds.dts
index c3839ae..333b349 100644
--- a/arch/powerpc/boot/dts/fsl/p3041ds.dts
+++ b/arch/powerpc/boot/dts/fsl/p3041ds.dts
@@ -82,10 +82,6 @@
 		ranges = <0x00000000 0xf 0x00000000 0x01008000>;
 	};
 
-	bportals: bman-portals@ff4000000 {
-		ranges = <0x0 0xf 0xf4000000 0x200000>;
-	};
-
 	qportals: qman-portals@ff4200000 {
 		ranges = <0x0 0xf 0xf4200000 0x200000>;
 	};
diff --git a/arch/powerpc/boot/dts/fsl/p3041si-post.dtsi b/arch/powerpc/boot/dts/fsl/p3041si-post.dtsi
index 26a6282..c8df07e 100644
--- a/arch/powerpc/boot/dts/fsl/p3041si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/p3041si-post.dtsi
@@ -47,6 +47,16 @@
 	alloc-ranges = <0 0 0x10 0>;
 };
 
+&qman_fqd {
+	compatible = "fsl,qman-fqd";
+	alloc-ranges = <0 0 0x10 0>;
+};
+
+&qman_pfdr {
+	compatible = "fsl,qman-pfdr";
+	alloc-ranges = <0 0 0x10 0>;
+};
+
 &lbc {
 	compatible = "fsl,p3041-elbc", "fsl,elbc", "simple-bus";
 	interrupts = <25 2 0 0>;
diff --git a/arch/powerpc/boot/dts/fsl/p4080ds.dts b/arch/powerpc/boot/dts/fsl/p4080ds.dts
index f747d6d..f72466a 100644
--- a/arch/powerpc/boot/dts/fsl/p4080ds.dts
+++ b/arch/powerpc/boot/dts/fsl/p4080ds.dts
@@ -82,10 +82,6 @@
 		ranges = <0x00000000 0xf 0x00000000 0x01008000>;
 	};
 
-	bportals: bman-portals@ff4000000 {
-		ranges = <0x0 0xf 0xf4000000 0x200000>;
-	};
-
 	qportals: qman-portals@ff4200000 {
 		ranges = <0x0 0xf 0xf4200000 0x200000>;
 	};
diff --git a/arch/powerpc/boot/dts/fsl/p4080si-post.dtsi b/arch/powerpc/boot/dts/fsl/p4080si-post.dtsi
index 3838146..612925d 100644
--- a/arch/powerpc/boot/dts/fsl/p4080si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/p4080si-post.dtsi
@@ -47,6 +47,16 @@
 	alloc-ranges = <0 0 0x10 0>;
 };
 
+&qman_fqd {
+	compatible = "fsl,qman-fqd";
+	alloc-ranges = <0 0 0x10 0>;
+};
+
+&qman_pfdr {
+	compatible = "fsl,qman-pfdr";
+	alloc-ranges = <0 0 0x10 0>;
+};
+
 &lbc {
 	compatible = "fsl,p4080-elbc", "fsl,elbc", "simple-bus";
 	interrupts = <25 2 0 0>;
diff --git a/arch/powerpc/boot/dts/fsl/p5020ds.dts b/arch/powerpc/boot/dts/fsl/p5020ds.dts
index bb5ed96..06da03f 100644
--- a/arch/powerpc/boot/dts/fsl/p5020ds.dts
+++ b/arch/powerpc/boot/dts/fsl/p5020ds.dts
@@ -82,10 +82,6 @@
 		ranges = <0x00000000 0xf 0x00000000 0x01008000>;
 	};
 
-	bportals: bman-portals@ff4000000 {
-		ranges = <0x0 0xf 0xf4000000 0x200000>;
-	};
-
 	qportals: qman-portals@ff4200000 {
 		ranges = <0x0 0xf 0xf4200000 0x200000>;
 	};
diff --git a/arch/powerpc/boot/dts/fsl/p5020si-post.dtsi b/arch/powerpc/boot/dts/fsl/p5020si-post.dtsi
index aecf5cc..1e4dda5 100644
--- a/arch/powerpc/boot/dts/fsl/p5020si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/p5020si-post.dtsi
@@ -47,6 +47,16 @@
 	alloc-ranges = <0 0 0x10000 0>;
 };
 
+&qman_fqd {
+	compatible = "fsl,qman-fqd";
+	alloc-ranges = <0 0 0x10000 0>;
+};
+
+&qman_pfdr {
+	compatible = "fsl,qman-pfdr";
+	alloc-ranges = <0 0 0x10000 0>;
+};
+
 &lbc {
 	compatible = "fsl,p5020-elbc", "fsl,elbc", "simple-bus";
 	interrupts = <25 2 0 0>;
diff --git a/arch/powerpc/boot/dts/fsl/p5040ds.dts b/arch/powerpc/boot/dts/fsl/p5040ds.dts
index ab02f92..1605c88 100644
--- a/arch/powerpc/boot/dts/fsl/p5040ds.dts
+++ b/arch/powerpc/boot/dts/fsl/p5040ds.dts
@@ -94,10 +94,6 @@
 		ranges = <0x00000000 0xf 0x00000000 0x01008000>;
 	};
 
-	bportals: bman-portals@ff4000000 {
-		ranges = <0x0 0xf 0xf4000000 0x200000>;
-	};
-
 	qportals: qman-portals@ff4200000 {
 		ranges = <0x0 0xf 0xf4200000 0x200000>;
 	};
diff --git a/arch/powerpc/boot/dts/fsl/p5040si-post.dtsi b/arch/powerpc/boot/dts/fsl/p5040si-post.dtsi
index 1a8ccbb..eb2414b 100644
--- a/arch/powerpc/boot/dts/fsl/p5040si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/p5040si-post.dtsi
@@ -47,6 +47,16 @@
 	alloc-ranges = <0 0 0x10000 0>;
 };
 
+&qman_fqd {
+	compatible = "fsl,qman-fqd";
+	alloc-ranges = <0 0 0x10000 0>;
+};
+
+&qman_pfdr {
+	compatible = "fsl,qman-pfdr";
+	alloc-ranges = <0 0 0x10000 0>;
+};
+
 &lbc {
 	compatible = "fsl,p5040-elbc", "fsl,elbc", "simple-bus";
 	interrupts = <25 2 0 0>;
-- 
2.7.4

